messages/training-checkout.txt

```diff
@@ -6,17 +6,31 @@ Thanks again for taking part in Software Carpentry (SWC) and Data Carpentry (DC)
+a) Look at how the existing exercises in that lesson are formatted, and format yours in the same way.
 
-Your change can be a fix to the existing material, a new exercise, or a diagram of some kind (preferably in a vector format like SVG, but we'll take anything).  We are *not* presently accepting material with new concepts: the lessons are full to bursting as it is, and our rule is that new stuff can only be added if we can identify something to take out.  (Note: refactorings that rearrange the order of material are also welcome.)
+b) If you're submitting by PR, please include discussion in the comments on the PR (*not* in the submitted content) about the purpose of the exercise.
 
-Your change doesn't have to be accepted in order for you to become an instructor - the point is to make sure that you understand the process and have met the people who review material.  If you want to do your final demonstration(s) in a language other than English, please let us know in advance so that we can find someone to sit in.
+Please note:
+
+- We do *not* want exercises or other material that introduce new concepts - our lessons are already full to bursting.
+
+- We accept submissions to DC for the purpose of becoming a SWC instructor.
+
+- Your change doesn't have to be accepted in order for you to become an instructor - the point is to make sure that you understand the process and have met the people who review material.
+
+2. Once you have submitted, pick a SWC lesson (if you want to become a SWC instructor) and/or a DC lesson (if you want to teach DC - if you want to teach both, pick one of each) and read through it carefully.  Once you've done that, sign up on http://pad.software-carpentry.org/lesson-discussion-2016 to take part in a discussion session.  Some of these sessions focus on specific lessons, while others are combined with our workshop debriefing sessions.  We're trying both formats for the next month or so to figure out which works best.
+
+If there aren't upcoming slots on the Etherpad that work for you, please add yourself to the top and we'll set something up.  You must do this step separately for each Carpentry, and you're expected to read through the lesson(s) carefully *before* your session and show up with questions.  If the session moderator (an experienced instructor) feels you haven't done this, you'll be invited to come back to a future session.
+
+3. Once you're through the discussion session(s), sign up for a 5-minute online demonstration lesson slot on http://pad.software-carpentry.org/teaching-demos (or add yourself to the top of the pad if none of the available slots works for you).  In that, we will ask you to teach a short segment from your chosen lesson - we'll pick the starting point on the day, so you will need to be familiar with the whole lesson.  If you want to do your final demonstration(s) in a language other than English, please let us know in advance so that we can find someone to sit in.
+
```

Comment:
change:
so you will need to be familiar with the whole lesson
to:
so you have to be prepared enough to be able to teach any given part of the lesson.